src/ansys/fluent/core/launcher/process_launch_string.py

```diff
@@ -54,6 +54,8 @@ def _build_fluent_launch_args_string(**kwargs) -> str:
                 argval = fluent_map[json_key]
             launch_args_string += v["fluent_format"].replace("{}", str(argval))
     addArgs = kwargs["additional_arguments"]
+    if addArgs:
```

Comment:
Above (existing) variable name is camel case. It's also abbreviated in such a way that its name changes meaning.

Reply:
Good catch, I agree, going to include the renaming of that symbol in this PR as well

Reply:
Commit 682b5112